doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md

```diff
@@ -27,7 +28,7 @@ Before building the C++ endpoint SDK, install the following components on your m
 1. Install g++, [CMake](https://cmake.org/download/), [Boost](http://www.boost.org/users/download/) and [SQLite3](https://sqlite.org/download.html):
 
    ```
-   sudo apt-get install g++ cmake libboost1.55-all-dev libsqlite3-0 libsqlite3-dev
+   sudo apt-get install g++ cmake python libboost-all-dev libsqlite3-0 libsqlite3-dev
```

Comment:
No, for Ubuntu 14.04 libboost-all-dev refers to libboost1.54-all-dev. But I'm convinced that [we can build cpp client](https://github.com/kaaproject/kaa/blob/release-0.10/client/client-multi/client-cpp/CMakeLists.txt#L393) using 1.54. Though, thx for the comment, I found that the patch also disables `avrogencpp` build. So, I'm going to change the guide: for Ubuntu 16.04 boost should be built from the source or we should create another patch which disables the tests build for Avro, but doesn't disable avrogencpp build. @Slash32, what option do you think fits better?